src/lib/platform/XWindowsPowerManager.cpp

```diff
@@ -32,16 +40,69 @@ XWindowsPowerManager::~XWindowsPowerManager()
 
 void XWindowsPowerManager::disableSleep() const
 {
-  if (!sleepInhibitCall(true, ArchSystemUnix::InhibitScreenServices::kScreenSaver) &&
-      !sleepInhibitCall(true, ArchSystemUnix::InhibitScreenServices::kSessionManager)) {
+  if (!sleepInhibitCall(true, XWindowsPowerManager::InhibitScreenServices::kScreenSaver) &&
+      !sleepInhibitCall(true, XWindowsPowerManager::InhibitScreenServices::kSessionManager)) {
     LOG((CLOG_WARN "failed to prevent system from going to sleep"));
   }
 }
 
 void XWindowsPowerManager::enableSleep() const
 {
-  if (!sleepInhibitCall(false, ArchSystemUnix::InhibitScreenServices::kScreenSaver) &&
-      !sleepInhibitCall(false, ArchSystemUnix::InhibitScreenServices::kSessionManager)) {
+  if (!sleepInhibitCall(false, XWindowsPowerManager::InhibitScreenServices::kScreenSaver) &&
+      !sleepInhibitCall(false, XWindowsPowerManager::InhibitScreenServices::kSessionManager)) {
     LOG((CLOG_WARN "failed to enable system idle sleep"));
   }
 }
+
+#ifndef __APPLE__
+bool XWindowsPowerManager::DBusInhibitScreenCall(InhibitScreenServices serviceID, bool state, std::string &error)
```

Comment:
While we're here...

```suggestion
bool XWindowsPowerManager::inhibitScreenCall(InhibitScreenServices serviceID, bool state, std::string &error)
```

I know this is moved code (someone else wrote it) but since `DBus` is an implementation, now might be a good opportunity to remove it from the function name, if you want to.